pyproject.toml

```diff
@@ -14,14 +14,20 @@ dependencies = [
     "uwsgi >=2.0",
     "uwsgitop >=0.11",
     "uwsgi-tools >=1.1.1",
-    "invenio-override ~=0.0.3"
+    "invenio-override ~=0.0.3",
+    "invenio-records-global-search ~=0.3.2",
+    "invenio-global-search >=0.2.0",
```

Comment:
move these packages to the invenio-override. https://github.com/sharedRDM/invenio-override/blob/72c861dbc452f84fb7abc29f9c8841fb714ee83d/setup.cfg#L38C1-L38C19